src/native/libs/CMakeLists.txt

```diff
@@ -128,6 +128,8 @@ if (CLR_CMAKE_TARGET_UNIX OR CLR_CMAKE_TARGET_BROWSER OR CLR_CMAKE_TARGET_WASI)
         add_compile_options(-Wno-cast-align)
         add_compile_options(-Wno-typedef-redefinition)
         add_compile_options(-Wno-c11-extensions)
```

Comment:
Could we instead use CMake's ability to check if the compiler supports this flag instead of also adding the "ignore unknown flags" option?

Reply:
```suggestion
        add_compile_options(-Wno-c11-extensions)

        check_c_compiler_flag(-Wc11-extensions COMPILER_SUPPORTS_W_C11_EXTENSIONS)
        if (COMPILER_SUPPORTS_W_C11_EXTENSIONS)
            add_compile_options(-Wno-c11-extensions)
        endif()

```